src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -130,10 +132,29 @@ class LovelacePanel extends LitElement {
 
   public firstUpdated() {
     this._fetchConfig(false);
-    // we don't want to unsub as we want to stay informed of updates
-    subscribeLovelaceUpdates(this.hass!.connection, () =>
-      this._lovelaceChanged()
-    );
+    if (this.urlPath === null) {
+      // we don't want to unsub as we want to stay informed of updates
+      subscribeLovelaceUpdates(this.hass!.connection, this.urlPath, () =>
+        this._lovelaceChanged()
+      );
+    } else if (!resourcesLoaded) {
+      // We only load resources from default config.
```

Comment:
I don't think this is necessary, as we handle duplicate resources in `loadLovelaceResources`, and besides that, we don't load new resources now when the config is changed.